### ISSUE TITLE
bqplot not working in Jupyter Lab on MyBinder

### ISSUE THREAD

**Issue by cornhundred** [bloomberg/bqplot#885]
**Describe the bug**
bqplot is not working in Jupyter Lab on MyBinder (and on my local machine). However, my custom widget, Clustergrammer2, is working in Jupyter Lab on MyBinder. 

**To Reproduce**
1. Go to https://mybinder.org/v2/gh/ismms-himc/clustergrammer2-notebooks/codex?urlpath=lab
2. Open Notebook: 6.0_CODEX_Example.ipynb
3. Run all cells
4. Cell 14 does not display widget and only shows "Error displaying widget". The Clustergrammer2 widget shows (see cell 13). The console shows the following errors:

<img width="654" alt="Screen Shot 2019-07-31 at 9 29 35 AM" src="https://user-images.githubusercontent.com/8352840/62215873-bc1be100-b375-11e9-9a54-a780b0863c52.png">

Note that bqplot is working on Jupyter Notebook on MyBinder (run 6.0 notebook to see bqplot work in Jupyter Notebook): https://mybinder.org/v2/gh/ismms-himc/clustergrammer2-notebooks/codex

**Expected behavior**
I exped to see the bqplot interactive widget in Jupyter Lab 

.
**Environment (please complete the following information):**
 - Operating System and Version: MacOS
 - Browser [e.g. Firefox, Safari] Chrome

**Additional context**
This code is on the `codex` branch of https://github.com/ismms-himc/clustergrammer2-notebooks/
I have the following steps in my [`postBuild`](https://github.com/ismms-himc/clustergrammer2-notebooks/blob/codex/postBuild) file: 
```
jupyter labextension install @jupyter-widgets/jupyterlab-manager
jupyter labextension install bqplot
```

**Comment by jtpio** [comment]
@cornhundred it looks like the `0.5.0-alpha.2` version is tagged as `latest` on [npm](https://www.npmjs.com/package/bqplot), which will be pulled when running the `jupyter labextension install bqplot` command.

What happens when pinning the version in `postBuild`?

`jupyter labextension install bqplot@0.4.6`

**Comment by arokem** [comment]
I am seeing similar issues with the current pypi version of bqplot (works in regular notebooks, but not in jupyterlab). 

What's the labextension version to use for the current released version of bqplot (0.12.16)? 

On the one hand, trying without pinned versions (as suggested in the README) does not work for me. On the other hand, pinning 0.4.6, as suggested here, didn't work either.

**Comment by ibdafna** [comment]
Just sanity checking - do you have the widgets manager JupyterLab extension
installed in addition to the bqplot one? If so, did you properly build
jupyter lab after installing these?

**Comment by arokem** [comment]
I think so. I am using repo2docker to automatically build the image, as described in this directory:

https://github.com/learning-2-learn/l2lhub-deployment/tree/staging/deployments/l2lhub/image

So, in `environment.yml` I have:  https://github.com/learning-2-learn/l2lhub-deployment/blob/staging/deployments/l2lhub/image/environment.yml#L30

And in `postBuild`, I have: 
https://github.com/learning-2-learn/l2lhub-deployment/blob/staging/deployments/l2lhub/image/postBuild#L3

But I am not sure what this means: 

> If so, did you properly build jupyter lab after installing these? 

Is there some additional step that needs to happen after this?

**Comment by ibdafna** [comment]
You're using bqplot 0.11.5 in your repo, which is quite an old version.
Same for the JS version. The latest backend version of bqplot is 0.12.6 and
0.5.6 for the front end. It looks like you're installing the extensions
correctly from what I've seen. Try running 'jupyter lab build' after the
extensions have been installed. Also try installing it locally with
virtualenv or conda (recommended) as you could have conflicts coming from
other packages in your Docker file.

**Comment by arokem** [comment]
Yep - adding a  "jupyter lab build" in my postBuild file did the trick (see https://github.com/learning-2-learn/l2lhub-deployment/pull/55/files). This should probably be in the README?
